Hardcoded path breaks Buildship

In /xtext-umbrella/gradle-composite/build.gradle there are hardcoded path entries to `../xtext-umbrella/releng/`

`xtext-umbrella` is not the project name but the git root directory which can be different depending on user configuration used during the Ommph setup.

Buildship stops resolution with:
```
Caused by: org.gradle.api.InvalidUserDataException: Project directory '/Users/dhuebner/git/xtext-umbrella/releng/gradle-composite' does not exist.
	at org.gradle.initialization.ProjectDirectoryProjectSpec.checkPreconditions(ProjectDirectoryProjectSpec.java:52)
	at org.gradle.initialization.AbstractProjectSpec.containsProject(AbstractProjectSpec.java:27)
	at org.gradle.initialization.DefaultSettingsLoader.findAndLoadSettings(DefaultSettingsLoader.java:51)
	at org.gradle.internal.composite.CompositeBuildSettingsLoader.findAndLoadSettings(CompositeBuildSettingsLoader.java:48)
	at org.gradle.initialization.NotifyingSettingsLoader.findAndLoadSettings(NotifyingSettingsLoader.java:33)
	at org.gradle.initialization.DefaultGradleLauncher.doBuildStages(DefaultGradleLauncher.java:147)
	at org.gradle.initialization.DefaultGradleLauncher.doBuild(DefaultGradleLauncher.java:119)
	... 44 more```

--- a/releng/gradle-composite/build.gradle
+++ b/releng/gradle-composite/build.gradle
@@ -8,7 +8,7 @@ task('generatePrefs') {
             if (!(new File(file, '.settings')).exists()) {
                 new File(file, '.settings').mkdir();
             }
-            (new File(file, '.settings/org.eclipse.buildship.core.prefs')).text = "connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)\nconnection.project.dir=../xtext-umbrella/releng/gradle-composite\nproject.path=\\:"+file.name+"\n"
+            (new File(file, '.settings/org.eclipse.buildship.core.prefs')).text = "connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)\nconnection.project.dir="+rootDir+"\nproject.path=\\:"+file.name+"\n"
                     
             file.listFiles().each { File fx -> 
                 if (fx.isDirectory() && new File(fx, "build.gradle").exists()) {
@@ -16,7 +16,7 @@ task('generatePrefs') {
                     if (!(new File(fx, '.settings')).exists()) {
                         new File(fx, '.settings').mkdir();
                     }
-                    (new File(fx, '.settings/org.eclipse.buildship.core.prefs')).text = "connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)\nconnection.project.dir=../../xtext-umbrella/releng/gradle-composite\nproject.path=\\:"+fx.name+"\n"
+                    (new File(fx, '.settings/org.eclipse.buildship.core.prefs')).text = "connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)\nconnection.project.dir="+rootDir+"\nproject.path=\\:"+fx.name+"\n"
                     }
 
                 }

